.github/workflows/build-deploy.yml

```diff
@@ -0,0 +1,61 @@
+name: Build & Deploy
+
+on:
+  push:
+    tags: [ '*' ]
+
+env:
+  GCP_LOCATION: europe-west1
+  GCP_PROJECT: reload-oliver-onboarding
+  GKE_CLUSTER: plann
+  RELEASE_TAG: ${{ github.ref_name }}
+
+jobs:
+  docker_build_push:
+    runs-on: ubuntu-latest
+    env:
+      APP_NAME: todo-app
+      REGISTRY: eu.gcr.io
+      IMAGE_PATH: ./docker/images/getting-started/app
+    steps:
+    - uses: actions/checkout@v3
+      name: Check out repo
+
+    - name: Docker Build & Push Achton
+      uses: mr-smithers-excellent/docker-build-push@v5.8
```

Comment:
Excellent that you got this working! :+1:

An alternate action to use could have been [the ones from Docker](https://github.com/marketplace/actions/build-and-push-docker-images), e.g.:

```yaml
   -
        name: Login to Docker Hub
        uses: docker/login-action@v2
        with:
          username: ${{ secrets.DOCKERHUB_USERNAME }}
          password: ${{ secrets.DOCKERHUB_TOKEN }}
      -
        name: Build and push
        uses: docker/build-push-action@v3
        with:
          push: true
          tags: user/app:latest
```